web/dashboard/src/views/Package.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div>
+    <h3>{{$route.query.ecosystem}}/{{$route.query.name}} @ {{$route.query.version}}</h3>
+    <h4>Files touched</h4>
+    <b-table :items="data.Files">
+    </b-table>
+    <h4>Commands</h4>
+    <b-table :items="data.Commands">
+    </b-table>
+    <h4>IPs</h4>
+    <b-table :items="getIpItems(data.IPs)">
+    </b-table>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Package',
+  components: {},
+  methods: {
+    async makeRequest() {
+      const resultsUrl = process.env.VUE_APP_ANALYSIS_RESULTS;
+      const response = await fetch(
+          resultsUrl + '/' +
+          this.$route.query.ecosystem + '/' +
```

Comment:
if there's a package to construct urls, we could use it.

Reply:
Don't think there's one builtin to Javascript..  Can revisit if this comes up more often.